src/Controls/samples/Controls.Sample/Startup.cs

```diff
@@ -45,7 +45,7 @@ public void Configure(IAppHostBuilder appBuilder)
 				})
 				.UseMauiServiceProviderFactory(true)
 				//.UseServiceProviderFactory(new DIExtensionsServiceProviderFactory())
-				.ConfigureServices((hostingContext, services) =>
+				.ConfigureServices(services =>
```

Comment:
I agree. This is just the simpler overload that I am using. The actual interface member has it still - since it is in the base. I just added an extension method to make things less intense for the new folks.